### PR TITLE
Sets up hapi server with views to serve current static files

### PR DESCRIPTION
**Pull request by shiryz**
Relates #81

### CODE REVIEW

**Comment by emilyb7** [comment]
all good from my point of view @shiryz , but if you only want to deploy to heroku for now, we don't need to deploy to heroku. You can deploy from any branch. Not sure what's best here...

File: src/server.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const Hapi = require('hapi');
+const Inert = require('inert');
+const Vision = require('vision');
+const Handlebars = require('handlebars');
+const fs = require('fs');
+
+const server = new Hapi.Server();
+
+// server.connection({
```
Comment (des-des) [inline src/server.js]: Delete

File: src/server.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+var tls = {
+    key : fs.readFileSync('./key.pem'),
+    cert : fs.readFileSync('./cert.pem')
+};
+
+server.connection({address: '0.0.0.0', port: process.env.PORT || 4000});
+
+server.register([Inert, Vision], (err) => {
+  if(err) throw err;
```
Comment (des-des) [inline src/server.js]: Good error throwing

File: src/server.js (hunk 1 above)
Comment (shiryz) [inline src/server.js]: The comment is for deployment

**Comment by des-des** [comment]
@shiryz confused got comment in email but lost it here 
image

**Comment by MHemsley** [comment]
Just to re-iterate in case this got lost, we’re looking to deploy to AWS in the longer term 


On 7 Mar 2017, at 18:53, Emily Bertwistle <notifications@github.com> wrote:


all good from my point of view 
@shiryz , but if you only want to deploy to heroku for now, we don't need to deploy to heroku. You can deploy from any branch. Not sure what's best here...
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, 
view it on GitHub, or 
mute the thread.







{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/CYPIAPT-LNDSE/goals-app","title":"CYPIAPT-LNDSE/goals-app","subtitle":"GitHub repository","main_image_url":"<a href="https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png" class="">https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png</a>","avatar_image_url":"<a href="https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png" class="">https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png</a>","action":{"name":"Open in GitHub","url":"<a href="https://github.com/CYPIAPT-LNDSE/goals-app" class="">https://github.com/CYPIAPT-LNDSE/goals-app</a>"}},"updates":{"snippets":[{"icon":"PERSON","message":"@emilyb7 in #104: all good from my point of view @shiryz , but if you only want to deploy to heroku for now, we don't need to deploy to heroku. You can deploy from any branch. Not sure what's best here..."}],"action":{"name":"View Pull Request","url":"<a href="https://github.com/CYPIAPT-LNDSE/goals-app/pull/104#issuecomment-284819569" class="">https://github.com/CYPIAPT-LNDSE/goals-app/pull/104#issuecomment-284819569</a>"}}}




  This email has been scanned for email related threats and delivered safely by Mimecast. For more information please visit http://www.mimecast.com

**Comment by des-des** [comment]
You can use `process.env` to toggle config based on whether you whether you are in production or dev..

**Comment by shiryz** [comment]
Going to close this PR, since we decided to deploy later, will open another one with changes.

**Pull request closed without merging**